neurons/miner.py

```diff
@@ -47,6 +63,38 @@ async def start(self, keypair: Keypair, port: int):
             logger.error(f"Failed to start miner: {str(e)}")
             raise
 
+    async def deregister_agent(self):
```

Comment:
I got a little bit confused about naming here, some things are register and others deregister?

Reply:
yeah this is a helper function exposed on the miner API for testing, you can mark your agent as isActive = false on the db and "deregsiter" as an active agent